com.hoptodesk.HopToDesk.json

```diff
@@ -0,0 +1,264 @@
+            "/share/gir-1.0"
+          ]
+        }
+      ]
+    },
+    {
+      "name" : "xdotool",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz",
+          "sha256" : "96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada",
+          "type" : "archive"
+        }
+      ],
+      "build-commands" : [
+        "make -j4 && PREFIX=./build make install",
+        "cp -r ./build/*  /app/"
+      ]
+    },
+    {
+      "name" : "libpam",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://github.com/linux-pam/linux-pam/releases/download/v1.6.1/Linux-PAM-1.6.1.tar.xz",
+          "sha256" : "f8923c740159052d719dbfc2a2f81942d68dd34fcaf61c706a02c9b80feeef8e",
+          "type" : "archive"
+        }
+      ],
+      "build-commands" : [
+        "./configure --prefix=$(pwd)/build",
+        "make -j4 && make install",
+        "cp -r ./build/*  /app/"
+      ]
+    },
+    {
+      "name" : "HopToDesk",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url": "https://www.hoptodesk.com/hoptodesk.deb",
+          "sha256": "6fb51d3d61fb82124d153a5f1f554acf72d6597152ed867ab17184b673935ee6",
+          "type" : "file"
+        },
+        {
+          "url": "https://gitlab.com/hoptodesk/hoptodesk/-/raw/main/res/logo.svg",
+          "sha256": "7107616c5649a5bd952479f02fb4619f764578fa8bcf54c4ea4bc5466244895b",
+          "type" : "file"
+        },
+        {
+          "path": "com.hoptodesk.HopToDesk.metainfo.xml",
+          "type": "file"
+        },
+        {
+          "path": "com.hoptodesk.HopToDesk.desktop",
+          "type": "file"
+        }
+      ],
+      "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/gtk-doc"
+      ],
+      "build-commands" : [
+        "bsdtar -zxvf hoptodesk.deb",
+        "tar -xvf ./data.tar.xz",
+        "cp -r ./usr/*  /app/",
+        "cp /app/lib/libsciter-gtk.so  /app/bin/",
+        "chmod 755 /app/bin/libsciter-gtk.so",
+        "mkdir -p /app/bin",
+        "mv com.hoptodesk.HopToDesk.desktop /app/share/applications/com.hoptodesk.HopToDesk.desktop",
+        "sed -i '/^Icon=/ c\\Icon=com.hoptodesk.HopToDesk' /app/share/applications/com.hoptodesk.HopToDesk.desktop",
+        "mv /app/share/icons/hicolor/128x128/128x128.png /app/share/icons/hicolor/128x128/com.hoptodesk.HopToDesk.png",
+        "for size in 16 24 32 48 64 128 256 512; do\n rsvg-convert -w $size -h $size -f png -o $size.png logo.svg\n install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/com.hoptodesk.HopToDesk.png\n done",
+        "mkdir -p /app/share/metainfo/ && cp com.hoptodesk.HopToDesk.metainfo.xml /app/share/metainfo/"
+      ]
+    }
+  ],
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=home",
```

Comment:
Why does it need home?

Reply:
It's used for file transferring between hosts